apps/website/src/components/show-and-tell/ShowAndTellEntry.tsx

```diff
@@ -169,10 +189,9 @@ export const ShowAndTellEntry = forwardRef<
               isPresentationView ? "scrollbar-none max-h-[66vh] pb-6" : ""
             }`}
           >
-            <div
-              className="alveus-ugc max-w-[1100px] hyphens-auto leading-relaxed md:text-lg xl:text-2xl"
-              dangerouslySetInnerHTML={{ __html: entry.text }}
-            />
+            <div className="alveus-ugc max-w-[1100px] hyphens-auto leading-relaxed md:text-lg xl:text-2xl">
+              {parse(entry.text, parseOptions)}
```

Comment:
I did wonder if we wanted to put an error boundary around this specifically, to avoid breaking the whole page if someone gets malformed HTML into the system? Or are we happy with the manual review process catching that?

Reply:
That does sound good.  While I don't think it's likely to happen but it does definitely not hurt.

Reply:
https://github.com/alveusgg/alveusgg/issues/216